use std::num::ParseIntError;

fn main() -> Result<(), ParseIntError>{
    getit()?;
    Ok(())
}

fn getit<E>() -> Result<(), E> {
    
    Ok(())
}